src/main/java/com/arpnetworking/clusteraggregator/models/StatusResponse.java

```diff
@@ -294,6 +294,7 @@ public Builder setSha(final String value) {
         /**
          * {@inheritDoc}
          */
+        @SuppressWarnings("deprecation") // The uid() method is apparently deprecated on UniqueAddress
```

Comment:
Fixed. They provide a replacement, I didn't see it before because the IDE was still loading the old source.